eos-tech-support/eos-enable-debian-apt

```diff
@@ -16,7 +16,7 @@ echo "Additionally, apt will only work after you have taken the usual steps of"
 echo "making it available (e.g. eos-convert-system, eos-dev-unlock, etc)"
 echo
 
-if [[ $# < 1 ]]; then
+if [[ $# -lt 1 ]] || [[ $1 == -* ]]; then
```

Comment:
Changes lgtm but I would probably change this conditional to `if [[ $# -ne 1 ]] || [[ $1 == -* ]];` to make sure only a single argument is allowed - although we only use the first argument so I will leave it at your discretion...

Reply:
Pushed this and one other cosmetic change; how does it look?